src/commands/extract/check.ts

```diff
@@ -0,0 +1,50 @@
+import type { BaseExtractOptions } from '../extract';
+import { relative } from 'path';
+import { Command } from 'commander';
+
+import { extractKeysOfFiles } from '../../extractor';
+import { WarningMessages, emitGitHubWarning } from '../../utils/warnings';
+
+type ExtractLintOptions = BaseExtractOptions;
+
+async function lintHandler(this: Command, filesPattern: string) {
+  const opts: ExtractLintOptions = this.optsWithGlobals();
+  const extracted = await extractKeysOfFiles(filesPattern, opts.extractor);
+
+  let warningCount = 0;
+  let filesCount = 0;
+  for (const [file, { warnings }] of extracted) {
+    if (warnings.length) {
+      warningCount += warnings.length;
+      filesCount++;
+
+      const relFile = relative(process.cwd(), file);
+      console.log('%s:', relFile);
```

Comment:
Calling some wrapped logger function would help us with future enhancing the console output.